src/giftCards/GiftCardsList/providers/GiftCardListProvider/hooks/useGiftCardListBulkActions.ts

```diff
@@ -0,0 +1,32 @@
+import { UseBulkActionsProps } from "@saleor/hooks/useBulkActions";
+import { useContext } from "react";
+
+import {
+  GiftCardsListConsumerProps,
+  GiftCardsListContext
+} from "../GiftCardListProvider";
+
+export type UseGiftCardListBulkActionsProps = UseBulkActionsProps &
+  Pick<GiftCardsListConsumerProps, "selectedItemsCount">;
+
+const useGiftCardListBulkActions = (): UseGiftCardListBulkActionsProps => {
```

Comment:
If these are separate hooks, doesn't it also make sense to split the providers as well? It's important we don't bloat the contexts too much

Reply:
Will make a separate pr for refactors 👍